gym_electric_motor/visualization/motor_dashboard.py

```diff
@@ -272,3 +272,7 @@ def _update(self):
         for fig in self._figures:
             fig.canvas.draw()
             fig.canvas.flush_events()
+
+    @property
```

Comment:
Please document this property. Furthermore, it is a common practice to put the properties on top of a class definition (after the docstring)